### ISSUE TITLE
Error when creating a EVC without Tag

### ISSUE THREAD

**Issue by italovalcy** [kytos/mef_eline#190]
According to the mef_eline API documentation [1], the Tag property is not required (which makes sense). However when you try to create an EVC without tag, you get the following error:

```
$ cat newcircuit-novlan.json
{
            "name": "EVC-Test",
            "uni_a": {
                "interface_id": "00:00:00:00:00:00:00:08:1"
            },
            "uni_z": {
                "interface_id": "00:00:00:00:00:00:00:02:1"
            }
}
$ curl -X POST -H "Content-Type: application/json" -d @newcircuit-novlan.json  http://localhost:8181/api/kytos/mef_eline/v2/evc/
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

From the server console:
```
kytos $> 2020-06-15 20:51:08,471 - ERROR [kytos.core.controller] (Thread-17423) Exception on /api/kytos/mef_eline/v2/evc/ [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "//var/lib/kytos/napps/kytos/mef_eline/main.py", line 126, in create_circuit
    evc = self._evc_from_dict(data)
  File "//var/lib/kytos/napps/kytos/mef_eline/main.py", line 597, in _evc_from_dict
    data = self._evc_dict_with_instances(evc_dict)
  File "//var/lib/kytos/napps/kytos/mef_eline/main.py", line 567, in _evc_dict_with_instances
    data[attribute] = self._uni_from_dict(value)
  File "//var/lib/kytos/napps/kytos/mef_eline/main.py", line 611, in _uni_from_dict
    tag = TAG.from_dict(tag_dict)
  File "/kytos/kytos/core/interface.py", line 44, in from_dict
    return cls(tag_dict.get('tag_type'), tag_dict.get('value'))
AttributeError: 'NoneType' object has no attribute 'get'
```

The environment is based on the following version (installed via python setup develop):
```
$ kytos napps list

Status |         NApp ID          |                                    Description
=======+==========================+==================================================================================
 [ie]  | kytos/flow_manager:2.2.2 | Manage switches' flows through a REST API.
 [ie]  | kytos/mef_eline:2.3.1    | NApp to provision circuits from user request
 [ie]  | kytos/of_core:1.4.1      | OpenFlow Core of Kytos Controller, responsible for main OpenFlow operations.
 [ie]  | kytos/pathfinder:2.2.2   | Keeps track of topology changes, and calculates the best path between two points.
 [ie]  | kytos/storehouse:1.3.0   | Persistence NApp with support for multiple backends
 [ie]  | kytos/topology:3.6.1     | Manage the network topology.

Status: (i)nstalled, (e)nabled
```

[1] https://github.com/kytos/mef_eline/blob/master/openapi.yml#L240-L249

**Comment by italovalcy** [comment]
Hi guys,

I'm still not able to create an EVC without tag, now the error is when kytos/mef_eline calls the storehouse and it tries to get a representation of the UNI as_dict:
```
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/dist-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/dist-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "//var/lib/kytos/napps/kytos/mef_eline/main.py", line 148, in create_circuit
    self.storehouse.save_evc(evc)
  File "//var/lib/kytos/napps/../napps/kytos/mef_eline/storehouse.py", line 89, in save_evc
    self.box.data[evc.id] = evc.as_dict()
  File "//var/lib/kytos/napps/../napps/kytos/mef_eline/models.py", line 328, in as_dict
    "uni_a": self.uni_a.as_dict(),
  File "/usr/local/lib/python3.6/dist-packages/kytos/core/interface.py", line 452, in as_dict
    'tag': self.user_tag.as_dict()}
AttributeError: 'NoneType' object has no attribute 'as_dict'
```

Can you please help with this issue?

Feel free to open a ticket on kytos/core and link it here.


Thanks, italo

**Comment by ajoaoff** [comment]
This error is caused by `kytos/kytos`. It has already been addressed in kytos/kytos#1080, but the PR has not yet been accepted.

**Comment by italovalcy** [comment]
Thank you, @ajoaoff! I just reviewed the PR you mentioned and I approved it. Let's just wait on @hdiogenes then.
